systemtest/scripts/results.json

```diff
@@ -0,0 +1 @@
+{"body":":heavy_exclamation_mark: **Systemtests Failed (no tests results are present)** :heavy_exclamation_mark:"}
```

Comment:
This probably should not be committed?

Reply:
oops. I remove it and add it to gitingore